### PR TITLE
Add tidb_auto_analyze_partition_batch_size variable

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11855
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Yisaer — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -702,6 +702,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `23:59 +0000`
 - This variable is used to restrict the time window that the automatic update of statistics is permitted. For example, to only allow automatic statistics updates between 1AM and 3AM, set `tidb_auto_analyze_start_time='01:00 +0000'` and `tidb_auto_analyze_end_time='03:00 +0000'`.
 
+### tidb_auto_analyze_partition_batch_size <span class="version-mark">New in v6.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Default value: `1`
```
Comment (hfxsd) [inline system-variables.md:709]: Add value range?

File: system-variables.md [hunk 2]
```diff
@@ -702,6 +702,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- This variable specifies the number of partitions that TiDB [automatically analyzes](/statistics.md#automatic-update) when analyzing a partitioned table (which means automatically collecting statistics on a partitioned table).
+- If the value of this variable is smaller than the number of partitions, TiDB automatically analyzes all partitions of the partitioned table in multiple batches. If the value of this variable is greater than the number of partitions, TiDB analyzes all partitions of the partitioned table at the same time.
```
Comment (hfxsd) [inline system-variables.md:711]:
```suggestion
- If the value of this variable is smaller than the number of partitions, TiDB automatically analyzes all partitions of the partitioned table in multiple batches. If the value of this variable is greater than or equal to the number of partitions, TiDB analyzes all partitions of the partitioned table at the same time.
```

File: system-variables.md [hunk 3]
```diff
@@ -702,6 +702,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- If the number of partitions of a partitioned table is far greater than this variable value and the auto-analyze takes a long time, you can increase the value of `tidb_auto_analyze_partition_batch_size` to reduce the time consumption.
```
Comment (hfxsd) [inline system-variables.md:712]:
```suggestion
- If the number of partitions of a partitioned table is far greater than this variable value and the auto-analyze takes a long time, you can increase the value of this variable  to reduce the time consumption.
```

File: system-variables.md (hunk 1 above)
Comment (TomShawn) [reply system-variables.md:709]:
```suggestion
- Default value: `1`
- Range: `[1, 1024]`
```

File: system-variables.md [hunk 4]
```diff
@@ -702,6 +702,16 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 
+### tidb_auto_analyze_partition_batch_size <span class="version-mark">New in v6.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Default value: `1`
+- Range: `[1, 1024]`
+- This variable specifies the number of partitions that TiDB [automatically analyzes](/statistics.md#automatic-update) when analyzing a partitioned table (which means automatically collecting statistics on a partitioned table).
+- If the value of this variable is smaller than the number of partitions, TiDB automatically analyzes all partitions of the partitioned table in multiple batches. If the value of this variable is greater than or equal to the number of partitions, TiDB analyzes all partitions of the partitioned table at the same time.
+- If the number of partitions of a partitioned table is far greater than this variable value and the auto-analyze takes a long time, you can increase the value of this variable  to reduce the time consumption.
```
Comment (hfxsd) [inline system-variables.md:713]:
```suggestion
- If the number of partitions of a partitioned table is far greater than this variable value and the auto-analyze takes a long time, you can increase the value of this variable to reduce the time consumption.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Pull request merged**